table/column.go

```diff
@@ -372,7 +372,8 @@ func getColDefaultValue(ctx sessionctx.Context, col *model.ColumnInfo, defaultVa
 			t := value.GetMysqlTime()
 			// For col.Version = 0, the timezone information of default value is already lost, so use the system timezone as the default value timezone.
 			defaultTimeZone := timeutil.SystemLocation()
-			if col.Version == model.ColumnInfoVersion1 {
+
+			if col.Version >= model.ColumnInfoVersion1 {
```

Comment:
Is this change related to the data race?

Reply:
It's my fault. 
Done.